java/rocksjni/writebatchhandlerjnicallback.cc

```diff
@@ -305,7 +305,7 @@ rocksdb::Status WriteBatchHandlerJniCallback::PutBlobIndexCF(uint32_t column_fam
   }
 }
 
-rocksdb::Status WriteBatchHandlerJniCallback::MarkBeginPrepare() {
+rocksdb::Status WriteBatchHandlerJniCallback::MarkBeginPrepare(bool) {
   m_env->CallVoidMethod(m_jcallback_obj, m_jMarkBeginPrepareMethodId);
```

Comment:
should it be changed to calling a non-void method?

Reply:
The void refers to the return value of the method I think.

I looked into this, and we could pass down the bool as an argument, so that it is exposed in the java api. I'm not very familiar with it though, so I'd prefer somebody more knowledgeable do it, if it's needed.

Reply:
makes sense. so for the time being can we assert that bool is always false so the java users do not mistakenly use the bool assuming that it would work.